### ISSUE TITLE
Can't see any members in new space with invited people

### ISSUE THREAD

**Issue by afranke** [vector-im/element-web#19055]
### Steps to reproduce

Create a new private space, invite people, go to Explore screen.

### What happened?

![image](https://user-images.githubusercontent.com/117241/133466085-3922be77-8032-48f6-8ca1-b2bf7e7a2142.png)

I expect to show members of this space, at least 1 (me), and I should have a way to open the member list to see invited people.

### Operating system

Fedora 34

### Browser information

Firefox 92

### URL for webapp

app.element.io

### Homeserver

matrix.org

### Have you submitted a rageshake?

No

**Comment by t3chguy** [comment]
Yeah so normally we show a facepile but if there's only 1 joined person (you) then there's nothing to show, we should have another way to open the members panel (beyond the context menu of the space on the left as that feels disconnected)

**Comment by daniellekirkwood** [comment]
Marking this as an enhancement as it's working "as expected". 

Thanks for the feedback though, we'll incorporate it into future design iterations!
